core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/MockBigQueryServiceFactory.scala

```diff
@@ -110,4 +116,10 @@ class MockGoogleBigQueryService(queryResponse: Either[Throwable, TableResult]) e
       case Right(results) => IO.pure(results)
     }
   }
+
+  override def createDataset(datasetName: String, labels: Map[String, String], aclBindings: Map[Acl.Role, Seq[(WorkbenchEmail, Entity.Type)]]): IO[DatasetId] = {
+    val dataset = DatasetInfo.newBuilder(datasetName).build().getDatasetId
+    IO.pure(dataset)
```

Comment:
tiny: Can this be one line? 
`IO.pure(DatasetInfo.newBuilder(datasetName).build().getDatasetId)`
Not a big deal tho - might be clearer as two